imap_processing/ultra/l0/ultra_utils.py

```diff
@@ -10,10 +10,13 @@ class PacketProperties(NamedTuple):
     logical_source: list  # List of logical sources
     addition_to_logical_desc: str  # Description of the logical source
     width: Union[int, None]  # Width of binary data (could be None).
-    block: Union[int, None]  # Number of values in each block (could be None).
-    # This is important for decompressing the images and
+    # The following are important for decompressing the images and
     # a description is available on page 171 of IMAP-Ultra Flight
     # Software Specification document (7523-9009_Rev_-.pdf).
+    block: Union[int, None]  # Number of values in each block (could be None).
+    image_panes: Union[int, None]  # number of images
+    pixel_window_rows: Union[int, None]  # number of rows in each image
+    pixel_window_columns: Union[int, None]  # number of columns in each image
```

Comment:
I realize there aren't many more packets left here now, but you adding these new fields with having to add a bunch of Nones everywhere it isn't needed made me think that it might be nice to have a `defaults=` in the named tuple set to None so you don't have to include them, or just switch over to a dataclass for these instead.